compaction/compaction.cc

```diff
@@ -68,6 +68,7 @@ static const std::unordered_map<compaction_type, sstring> compaction_types = {
     { compaction_type::Reshard, "RESHARD" },
     { compaction_type::Upgrade, "UPGRADE" },
     { compaction_type::Reshape, "RESHAPE" },
+    { compaction_type::Splitting, "SPLITTING" },
```

Comment:
To maintain the same sense, we should use the infinitive form (SPLIT) rather than the present participle.